pkg/k8s/apps/translate.go

```diff
@@ -532,16 +532,14 @@ func TranslateContainerSecurityContext(c *apiv1.Container, s *model.SecurityCont
 
 	if s.RunAsUser != nil {
 		c.SecurityContext.RunAsUser = s.RunAsUser
-		if *s.RunAsUser == 0 {
-			c.SecurityContext.RunAsNonRoot = &falseBoolean
```

Comment:
I think you can remove the `falseBoolean` variable:
https://deepsource.io/gh/okteto/okteto/run/69a9ca14-39ce-4acf-82a8-f5c59085ba3c/go/SCC-U1000

Reply:
Thanks for approval! I have just updated this with the variable removed and an extra test case for coverage. I will try find some time tomorrow to fix the last coverage warning.